ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-master.yaml

```diff
@@ -0,0 +1,48 @@
+base_images:
+  cli:
+    cluster: https://api.ci.openshift.org
+    name: "4.5"
+    namespace: ocp
+    tag: cli
+binary_build_commands: make build
+build_root:
+  project_image:
+    dockerfile_path: openshift-ci/build-root/Dockerfile
+images:
+- from: bin
```

Comment:
You need here to specify a `dockerfile_path` and inject the `bin` in inputs as you do with `cli` image.

Reply:
Thank you! Updated it.